### PR TITLE
updates to shannox

### PR DESCRIPTION
**Pull request by sunshitwowsucks**

### CODE REVIEW

File: sql/updates/world/1_10_28_16_world.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+delete from spell_script_names where spell_id in(101111, 99837, 99838);
```
Comment (Keital) [inline sql/updates/world/1_10_28_16_world.sql]: typo `spell_script_names`

File: sql/updates/world/1_10_28_16_world.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES 
+(101111, 'spell_shannox_riplimb_dogged_determination'),
+(99837, 'spell_shannox_crystal_prison_trap'),
+ (99838, 'spell_shannox_immolation_trap');
+
+update creature_template set flags_extra = 128 where entry = 53393;
```
Comment (Keital) [inline sql/updates/world/1_10_28_16_world.sql]: same here

File: sql/updates/world/1_10_28_16_world.sql (hunk 1 above)
Comment (sunshitwowsucks) [inline sql/updates/world/1_10_28_16_world.sql]: it doesn't matter for this.

File: sql/updates/world/1_10_28_16_world.sql (hunk 2 above)
Comment (sunshitwowsucks) [inline sql/updates/world/1_10_28_16_world.sql]: nor here

Comment (Keital) [inline sql/updates/world/1_10_28_16_world.sql]: and here

**Pull request merged**